artifacts/scripts/util.sh

```diff
@@ -853,6 +854,13 @@ update-deps-in-gomod() {
     ensure-clean-working-dir
 }
 
+function fixAmbiguousImports() {
+   # ref: https://github.com/kubernetes/publishing-bot/issues/304
+   # TODO(nikhita): remove this when k/k drops or bumps
+   # cloud.google.com/go to a version > v0.105.0
+   go get cloud.google.com/go@v0.97.0
```

Comment:
Hmm, I'm not sure we should do this… doesn't this change the versions the module is requiring?

Reply:
> doesn't this change the versions the module is requiring

Yes, but it ends up being a no-op for repos (e.g kube-aggregator) that don't need to `require` `cloud.google.com/go` because `go mod tidy` will remove it from go.mod.

For repos that already `require` it (e.g. apiserver), they are already at v0.97.0 so it ends up being a no-op again.

Reply:
This still seems like something that shouldn't happen at this level, and will get stale or force the wrong version eventually on some branch